resources/aws-elb/description.yaml

```diff
@@ -1,73 +1,70 @@
----
 apiVersion: v1
 kind: Description
-app: "AWS ELB"
+app: AWS ELB
```

Comment:
are you sure removing quotes is fine?

Reply:
Yes, it is fine and it is correctly parsed by both pytest and the backend.
Actually, quotes are removed by the pyyaml, as there is no disambiguaty with other type.